crates/bevy_tasks/src/lib.rs

```diff
@@ -33,5 +33,16 @@ pub mod prelude {
     };
 }
 
-pub use num_cpus::get as logical_core_count;
-pub use num_cpus::get_physical as physical_core_count;
+use std::num::NonZeroUsize;
+
+/// Gets the logical CPU core count available to the current process.
+///
+/// This is identical to [`std::thread::available_parallelism`], except
+/// it will return a default value of 1 if it internally errors out.
+///
+/// This will always return at least 1.
+pub fn available_parallelism() -> usize {
+    std::thread::available_parallelism()
+        .map(NonZeroUsize::get)
+        .unwrap_or(1)
```

Comment:
This review comment isn't resolved yet. I'm fine with either adding the warning if it fails or a reasonable argument why it shouldn't be added.

Reply:
This was mostly to keep the behavior the same as it was before (as with num_cpus). Also given that this can be called repeatedly, it could easily spam the logs made.